app/src/main/java/com/eveningoutpost/dexdrip/models/JoH.java

```diff
@@ -780,6 +780,33 @@ public static String niceTimeScalar(long t) {
         return qs((double) t, 0) + " " + unit;
     }
 
+    public static String niceTimeScalarEn(long t) { // English only - Please do NOT translate
+        String unit = "second";
+        t = t / 1000;
+        if (t != 1) unit = "seconds";
+        if (t > 59) {
+            unit = "minute";
+            t = t / 60;
+            if (t != 1) unit = "minutes";
+            if (t > 59) {
+                unit = "hour";
+                t = t / 60;
+                if (t != 1) unit = "hours";
+                if (t > 24) {
+                    unit = "day";
+                    t = t / 24;
+                    if (t != 1) unit = "days";
+                    if (t > 28) {
```

Comment:
why t > 28?! should it be t > 7 
Please check if this is an error

Reply:
Even though it may look like I am adding this code, I am not.
I have copied an existing code, removed translation strings from it, and replaced them with English text.

Your questions should be posed with respect to the original existing code here: 
https://github.com/NightscoutFoundation/xDrip/blob/8da5e5d094478e266011f60ac4041042e0178754/app/src/main/java/com/eveningoutpost/dexdrip/models/JoH.java#L783-L809  

If your questions and suggestions are accepted, the original code as well as my copy can be modified.
If your suggestions are not accepted for the original code, I don't see why they would only apply to my copy.

Reply:
Thanks

The lead developer will see your review and my response to it.  Hopefully he can make a decision.

Thanks to your comment, I realized that I needed to clarify that I am just copying an existing code and am not creating code from scratch.  So, I appreciate your review.